Integrations/GoogleKeyManagementService/GoogleKeyManagementService.py

```diff
@@ -0,0 +1,854 @@
+
+
+def main():
+    COMMANDS = {
+        f'{INTEGRATION_COMMAND_NAME}create-key': (create_crypto_key_command, ['Project-Admin', 'KMS-Admin']),
+
+        f'{INTEGRATION_COMMAND_NAME}symmetric-decrypt': (symmetric_decrypt_key_command, ['Decrypter',
+                                                                                         'Encrypter/Decrypter',
+                                                                                         'Project-Admin']),
+
+        f'{INTEGRATION_COMMAND_NAME}symmetric-encrypt': (symmetric_encrypt_key_command, ['Encrypter',
+                                                                                         'Encrypter/Decrypter',
+                                                                                         'Project-Admin']),
+
+        f'{INTEGRATION_COMMAND_NAME}get-key': (get_key_command, ['Project-Admin', 'KMS-Admin']),
+
+        f'{INTEGRATION_COMMAND_NAME}update-key': (update_key_command, ['Project-Admin', 'KMS-Admin']),
+
+        f'{INTEGRATION_COMMAND_NAME}destroy-key': (destroy_key_command, ['Project-Admin', 'KMS-Admin']),
+
+        f'{INTEGRATION_COMMAND_NAME}restore-key': (restore_key_command, ['Project-Admin', 'KMS-Admin']),
+
+        f'{INTEGRATION_COMMAND_NAME}disable-key': (disable_key_command, ['Project-Admin', 'KMS-Admin']),
+
+        f'{INTEGRATION_COMMAND_NAME}enable-key': (enable_key_command, ['Project-Admin', 'KMS-Admin']),
+
+        f'{INTEGRATION_COMMAND_NAME}list-keys': (list_keys_command, ['Project-Admin', 'KMS-Admin']),
+
+        f'{INTEGRATION_COMMAND_NAME}asymmetric-encrypt': (asymmetric_encrypt_command, ['Encrypter',
+                                                                                       'Encrypter/Decrypter',
+                                                                                       'Project-Admin']),
+
+        f'{INTEGRATION_COMMAND_NAME}asymmetric-decrypt': (asymmetric_decrypt_command, ['Decrypter',
+                                                                                       'Encrypter/Decrypter',
+                                                                                       'Project-Admin'])
+    }
+
+    command = demisto.command()
+    LOG(f'{INTEGRATION_NAME}: command is {command}')
+    try:
+        client = Client(demisto.params())
+
+        if command == 'test-module':
+            test_function(client)
+            demisto.results('ok')
+
+        if command not in COMMANDS:
+            raise NotImplementedError(f'Command "{command}" is not implemented.')
+
+        cmd_func = str(COMMANDS.get(command)[0])
+        cmd_role = list(COMMANDS.get(command)[1])
+
+        if client.role not in cmd_role:
+            raise Exception(f"Your Service Account Role does not permit the use of {command} command.")
+
+        results = cmd_func(client, demisto.args())  # type: ignore
+        if results:
```

Comment:
you should always return results (otherwise the command will never finish

Reply:
done